tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant_happy_paths.py

```diff
@@ -330,6 +336,144 @@ def test_sql_happy_path_onboarding_data_assistant(
     assert results.success is False
 
 
+@pytest.mark.integration
+@pytest.mark.slow  # 6.54 seconds
+def test_sql_happy_path_onboarding_data_assistant_null_column_quantiles_metric_values(
+    sa,
+    empty_data_context,
+) -> None:
+    context: DataContext = empty_data_context
+
+    db_file = file_relative_path(
+        __file__,
+        os.path.join(
+            "..",
+            "..",
+            "test_sets",
+            "taxi_yellow_tripdata_samples",
+            "sqlite",
+            "yellow_tripdata.db",
+        ),
+    )
+
+    # This actually works with any SQLAlchemy-compatible backend; the naming will reflect this as part of near-term fix.
```

Comment:
Could you link this ticket? https://greatexpectations.atlassian.net/browse/GREAT-1400

Reply:
@billdirks We are not supposed to put JIRA links into the OSS codebase.

Reply:
Oh, I didn't we weren't suppose to do this. Thanks.